tests/51media/01unicode.pl

```diff
@@ -167,7 +167,13 @@ sub test_using_client
 
    get_media( $client, $content )->then( sub {
       my ( $cd_params ) = @_;
-      assert_eq( $cd_params->{'filename*'}, "utf-8''$FILENAME_ENCODED", "filename*" );
+
+      if (exists( $cd_params->{'filename'} )) {
+         assert_eq( $cd_params->{'filename'}, "utf-8\"$FILENAME\"", "filename" );
```

Comment:
fyi single-quotes can be elided on words within `{}`:

```suggestion
         assert_eq( $cd_params->{filename}, "utf-8\"$FILENAME\"", "filename" );
```

(`filename*` needs them because otherwise the * gets interpreted as an operator)

it doesn't matter though.